src/main/webapp/script.js

```diff
@@ -112,6 +112,7 @@ function loadHeatmap(data) {
 
 function convertGriddedDataToWeightedPoints(griddedData) {
     const dataGrid = griddedData.data;
+    console.log(dataGrid);
```

Comment:
ifs this for debugging? or part of code

Reply:
actually, may be useful for debugging on my end too? 😅 but i meant will this be here for the production code?

Reply:
Oops, just debugging

Reply:
we can keep it in for now?  might use it